### DISCUSSION TITLE
DreamBerd is back

### DISCUSSION THREAD

**Discussion by TodePond** [TodePond/GulfOfMexico#479] [Announcements]
[Recently](https://github.com/TodePond/DreamBerd/releases/tag/v0.0), we ran out of money.

However, I just realised that I counted our investments wrong. User Joshix didn't just donate 69 *cents*. They donated 69 [*euros*](https://github.com/TodePond/DreamBerd/tree/main/docs/investment/pre-seed#liberapay-investors)!

> DreamBerd is saved!

## Our new drive into academia

Since [conquering hacker news](https://github.com/TodePond/DreamBerd/issues/32) (a vile website), we've set our sites on a new target: Academia.

It is our goal to get DreamBerd mentioned in as many academic papers and conferences as possible. If you work in academia, or you know someone who does, **mention DreamBerd**!

The current list of DreamBerd appearances is as follows:

- [Cellpond: Spatial programming without escape](https://www.youtube.com/watch?v=cBYudbaqHAk&t=6704s).
- That's it.

Due to the slow place of academia, we don't expect this list to grow any time soon.

## Influencers

No, we're not interested in any influencers covering Dreamberd (unless you work in academia).

But if you really insist on covering it... You <ins>must</ins> cover the contributing section in your coverage. We've added a [note](https://github.com/TodePond/DreamBerd/tree/main#contributing) to our docs to cover this.

**If you notice an influencer or content-creator cover DreamBerd without mentioning it, please let them know in their chat and comments!**

<br>

Thank you for tuning into this week's release notes. See you next week.

<hr /><em>This discussion was created from the release <a href='https://github.com/TodePond/DreamBerd/releases/tag/v1.0'>DreamBerd is back</a>.</em>

**Comment by amytimed** [comment]
bro reacted to his own discussion with launch emoji to make it seem more active

**Reply by amytimed** [reply]
so what? after stalking your profile, you did the exact same thing on your gh issues! shame on you

**Reply by mybearworld** [reply]
Wow, that is terrible behavior. Who would do that?!

**Comment by gabrielchl** [comment]
ok, it's become apparent that we need a way to improve the financial security of DreamBerd,
i propose that we introduce the Installation Fee System ™, which works as described below:

1st time using the installer installer: FREE!
2nd time using the installer installer or update: $1
3rd time using the installer installer or update: $2
4th time using the installer installer or update: $4
5th time using the installer installer or update: $8
...

we'll also hire a PR firm to promote how this innovative system would be beneficial our developers

we should be rich in no time  🚀

**Reply by MaddyGuthridge** [reply]
![image](https://github.com/TodePond/DreamBerd/assets/31365175/725fb275-11c4-46e3-8d84-118e6e19facb)

**Reply by mybearworld** [reply]
That is a good idea. DreamBerd should utilize the superior currency, rice.

**Comment by NotAShelf** [comment]
why are we not investing into berdcoin?

**Reply by theprogrammingbird** [reply]
no idea

**Comment by chochelius** [comment]
I've said it before and I'll said it again, DreamBerd is the future of IT in all it's fields. Whoever isn't on board will get left behind and is, without any doubt, a sucker. 
DreamBerd now, DreamBerd forever and DreamBerd until my last breath.

**Comment by mybearworld** [comment]
you just lost the game
